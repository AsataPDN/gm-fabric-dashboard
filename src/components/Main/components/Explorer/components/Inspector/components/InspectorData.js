import styled from "styled-components";

import { spacingScale } from "style/styleFunctions";

const InspectorData = styled.ul`
  flex: 1 1 100%;
  margin: 0 auto;
  -webkit-overflow-scrolling: touch;
  overflow-y: scroll;
  padding: 0 ${spacingScale(1)};
  width: 100%;
  word-break: break-all;
`;

export default InspectorData;
