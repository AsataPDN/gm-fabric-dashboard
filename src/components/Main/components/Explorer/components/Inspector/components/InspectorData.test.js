import React from "react";
import { shallow } from "enzyme";
import InspectorData from "./InspectorData";

describe("InspectorData", () => {
  it("matches snapshot", () => {
    const aInspectorData = shallow(<InspectorData />);
    expect(aInspectorData).toMatchSnapshot();
  });
});
