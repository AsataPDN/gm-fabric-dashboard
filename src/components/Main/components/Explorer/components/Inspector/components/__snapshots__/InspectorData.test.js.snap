// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InspectorData matches snapshot 1`] = `
.c0 {
  -webkit-flex: 1 1 100%;
  -ms-flex: 1 1 100%;
  flex: 1 1 100%;
  margin: 0 auto;
  -webkit-overflow-scrolling: touch;
  overflow-y: scroll;
  padding: 0 8px;
  width: 100%;
  word-break: break-all;
}

<ul
  className="c0"
/>
`;
